apps/web/src/app/dashboard/[name]/overview/page.tsx

```diff
@@ -14,23 +14,53 @@ import { Toggle } from "~/components/ui/toggle";
 import projectMockData from "~/lib/mock/organizationOverview";
 import { getOwnOrganizations } from "~/server/actions/organization";
 
-export default function Dashboard() {
-  // TODO: replace mocks with db queries
-  const { data: _organizations } = useQuery({
-    queryKey: ["organizations"],
-    queryFn: () => getOwnOrganizations(),
+type DashboardOverviewProps = {
+  params: {
+    name: string;
+  };
+};
+
+export default function DashboardOverviewPage({
+  params,
+}: DashboardOverviewProps) {
+  const { data: organizations, isLoading } = useQuery({
+    queryKey: ["organizations", params.name],
+    queryFn: async () => {
+      const [organizations, err] = await getOwnOrganizations();
+      if (err !== null) {
+        console.error(err);
+      }
+
+      if (organizations === null) {
+        return [];
+      }
```

Comment:
Returning an empty array is not what I meant by "Add error handling". Also if an `error !== null` than `organizations === null`. This check is unnecessary and we can add this in the `error !== null` block.

Reply:
fixed and added redirect and toast on error